modules/examples/src/main/java/com/ibm/cloud/platform_services/enterprise_management/v1/EnterpriseManagementExamples.java

```diff
@@ -199,6 +199,7 @@ public static void main(String[] args) throws Exception {
       String importAccountId = "<accountid_to_be_imported>";
 
       System.out.println("importAccountToEnterprise() result:");
+
```

Comment:
No need for a label when we'll be displaying the status code after the "end" comment as we are below.